Lib/test/test_ast.py

```diff
@@ -243,6 +243,12 @@ def to_tuple(t):
 
 class AST_Tests(unittest.TestCase):
 
+    def _is_ast_node(self, name, node):
+        if not (name[0].isupper() and isinstance(node, type)):
+            return False
+        if node is not ast.AST and issubclass(node, ast.AST):
+            return True
```

Comment:
In this way, this is a bit deceiving because both `ifs` can be false and then the function returns None.

Reply:
What is wrong with returning `None`, it is still falsy. But for clarifying, I can add a return False.

Reply:
Oh my bad. I still dont understand what is the purpose of that test so I might did refactoring wrong, sorry. I'll try to do it again